### ISSUE TITLE
Handling additional arguments

### ISSUE THREAD

**Issue by nickbabcock** [crate-ci/resources#10]
I've been executing with `--all` on cargo components even if the workspace just contains a single package

```
cargo clippy --all
cargo fmt --all
```

Currently `clippy.yml`, `rustfmt.yml`, `warnings.yml` don't support `--all`. While I'd vote for `--all` to be the default behavior, this problem is a subset of handling additional arguments. For instance, maybe I want to execute clippy and fmt with a verbose flag, so maybe a one size fits all won't work here.

I was curious if there has been any thought given to this use case.

**Comment by epage** [comment]
I agree that `--all` should be added.  One of the questions though is if we should bother providing these templates.

**Comment by nickbabcock** [comment]
> One of the questions though is if we should bother providing these templates.
True. I suppose they shouldn't be the priority for the moment.
